### PR TITLE
fix: scripts to run tee-worker ts tests on a SGX machine

### PR DESCRIPTION
**Pull request by wli-pro**

### CODE REVIEW

File: tee-worker/README.md [hunk 1]
```diff
@@ -42,6 +42,22 @@ make
 ### Integration Tests
 See [docker/README.md](docker/README.md)
 
+### Quickly launch parachain, tee worker and run ts-test against tee-worker
+
+```
+source "$HOME/.cargo/env"
```
Comment (kziemianek) [inline tee-worker/README.md]: you only need to run it once after rustup installation so I think it is not necessary to include it here.

File: tee-worker/README.md [hunk 2]
```diff
@@ -42,6 +42,22 @@ make
...
+source /opt/intel/sgxsdk/environment
```
Comment (kziemianek) [inline tee-worker/README.md]: I suggest adding comment that location of this file might differ across environments

File: tee-worker/README.md [hunk 3]
```diff
@@ -42,6 +42,22 @@ make
...
+# run clean if only SGX_MODE change
+make clean
+make SGX_MODE=SW
+./local-setup/launch.py local-setup/github-action-config-one-worker.json
+# run script below if container started
```
Comment (kziemianek) [inline tee-worker/README.md]: i'm not sure about the rest part because I stopped once I checked the logs

File: tee-worker/README.md (hunk 1 above)
Comment (wli-pro) [reply tee-worker/README.md]: added it as comment in case new beginner doesn't have it in bash profile.

File: tee-worker/README.md (hunk 2 above)
Comment (wli-pro) [reply tee-worker/README.md]: added the comment now.

File: tee-worker/README.md [hunk 4]
```diff
@@ -42,6 +42,24 @@ make
...
+source /opt/intel/sgxsdk/environment
+# run clean if only SGX_MODE change
+make clean
+make SGX_MODE=SW
+./local-setup/launch.py local-setup/github-action-config-one-worker.json
+# run the ts test script below if container started
+cd  ./tee-worker/ts-tests
+yarn
+yarn run test-identity:local
+# other ts test
```
Comment (kziemianek) [inline tee-worker/README.md]: I don't know what you mean by this comment 🙃

**Review by kziemianek — APPROVED** [review]

**Comment by Kailai-Wang** [comment]
Thank you! Could you please shift this section to the very beginning of this README - before the original Integritee doc (so that we know it's from us)

This README needs to be better structured as well - but there's another issue for it

**Pull request merged**